### DISCUSSION TITLE
Technology preview: FLARM integration

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enroute#310] [Team Posts]
Dear all,

a few minutes ago I pushed a technology preview for our app to Google Play. The app can now connect to your aircraft's traffic receiver (typically a FLARM device) and show nearby traffic on the moving map. Open the menu and go to **Information/Traffic Reveicer** to try it out.

Please send me your feedback! Reach me by e-mail, GitHub, or via any other channel.

- Does the feature work for you at all? Do you run into problems?
- If it works, how do you like the map design?
- What traffic receiver do you use?
- Was the setup difficult? Did you find the help texts helpful? Did you even find the help texts?

Best,

Stefan.

**Comment by Ivan081** [comment]
Ho why not show RealTime position of enroute user? Each device when in fly
send to server his position.




Il ven 19 feb 2021, 07:32 Stefan Kebekus <notifications@github.com> ha
scritto:
